core/src/js/components/sort-dropdown.component.ts

```diff
@@ -0,0 +1,113 @@
+import {
+	AfterViewInit,
+	ChangeDetectionStrategy,
+	ChangeDetectorRef,
+	Component,
+	ElementRef,
+	EventEmitter,
+	Input,
+	Output,
+	ViewRef,
+} from '@angular/core';
+import { IOption } from 'ng-select';
+
+@Component({
+	selector: 'sort-dropdown',
+	styleUrls: [`../../css/global/scrollbar.scss`, `../../css/component/sort-dropdown.component.scss`],
```

Comment:
Here I would reference the .scss of the filter-dropdown component, and use a new file only for the style that is specific to that new component. This would make it easier to understand the differences between the two